librz/util/file.c

```diff
@@ -1032,16 +1030,16 @@ RZ_API int rz_file_mkstemp(RZ_NULLABLE const char *prefix, char **oname) {
 		prefix = "rz";
 	}
 #if __WINDOWS__
-	LPTSTR name = NULL;
-	LPTSTR path_ = rz_sys_conv_utf8_to_win(path);
-	LPTSTR prefix_ = rz_sys_conv_utf8_to_win(prefix);
+	LPWSTR name = malloc(sizeof(WCHAR) * (MAX_PATH + 1));
+	LPWSTR path_ = rz_utf8_to_utf16(path);
```

Comment:
What about using `wpath` or `pathw` instead or the hardly noticeable `_` suffix at the end? It might cause a confusion during the quick look.